app/src/main/res/menu/menu_main.xml

```diff
@@ -58,6 +58,11 @@
         android:showAsAction="never"
         android:title="@string/create_new_folder"
         app:showAsAction="never" />
+    <item
+        android:id="@+id/access_more_media"
+        android:showAsAction="never"
+        android:title="Allow access to more media"
```

Comment:
what is this for? It should have a proper string, not a hardcoded one

Reply:
Yeah, I need to define a proper string, just need to finalize the wording.
This is related to new media permissions - users can now select only a subset images they want to allow our app access to. Based on guidelines in https://developer.android.com/about/versions/14/changes/partial-photo-video-access we are supposed to let users manually allow access to more media at any point (without constantly prompting them on startup), so this option just displays the permission popup to the user to let them add more images or allow all.